projects/packages/publicize/src/class-publicize-script-data.php

```diff
@@ -161,6 +162,41 @@ public static function get_social_settings() {
 		return array(
 			'socialImageGenerator' => $settings->get_image_generator_settings(),
 			'utmSettings'          => $settings->get_utm_settings(),
+			// TODO remove this and use the settings via publicize package.
+			'socialPlugin'         => array(
+				'social_notes_config' => array(),
+			),
```

Comment:
Since `socialPlugin` field came from Social plugin and the corresponding UI components in Social plugin expected the same. Now that everything is in publicize package, we should have these config initial state passed from publicize package. But, for now let us add the property here to avoid making too many changes in this PR. In a follow up we can update that property to be something like `socialConfig` or something similar or may be move that notes config up a level.

Reply:
`jetpackSocialInitialState` is no longer used and it should be cleaned up as a part of https://github.com/Automattic/jetpack-reach/issues/763

Reply:
I added this in the initial versin of this PR where I enabled the page for both Social and Jetpack plugin, just to test if there are any gotchas and it resulted in JS errors when only Jetpack was active and thus I added this fallback. Now that we only show the page only when Social is active, we can safely remove it and migrate that settings as agreed when we turn it on for Jetpack plugin.

I have removed it for now.